### DISCUSSION TITLE
Incorrect error message when upvoting a discussion

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [community/community#23230] [Discussions]
<p>When upvoting a discussion, the error that is shown is “You must be logged in to vote”, when I am already logged in.</p>
<p>The error should either be “You have already voted this discussion” or a successful up voting.</p>
<p><img src="https://user-images.githubusercontent.com/1639302/181075672-c84acce3-e5f0-47bd-a5e6-78ecc7a2c852.gif" alt="CleanShot 2021-01-04 at 19.45.01" data-base62-sha1="jnFoPvgV1ZikXO6a3vhxkfu5pwt" width="690" height="159" class="animated"></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Thanks for letting us know. It was reported <a href="https://github.community/t/confusing-upvote-button-on-discussions/151980">here</a> so we already have an issue open for the team.</p>
